testdata/config/azure_key_lfu.yaml

```diff
@@ -20,6 +20,7 @@ file_cache:
   max-size-mb: 2048
   allow-non-empty-temp: true
   cleanup-on-start: true
+  direct-read: true
```

Comment:
If we enable direct read in all these test files, then all our pipeline tests will not hit the file-cache. May be this will end up overlooking some potential bugs in file-cache. Does this flag make good amount of difference in perf for resnet50 ? if so we can create another file for resNet50 test and leave these others as is.